airbyte-integrations/connectors/source-github/source_github/streams.py

```diff
@@ -802,3 +802,75 @@ class PullRequestCommentReactions(ReactionStream):
     """
 
     parent_entity = ReviewComments
+
+
+# Action streams
+
+
+class Workflows(GithubStream):
+    """
+    Get all workflows of a GitHub repository
+    API documentation: https://docs.github.com/en/rest/reference/actions#workflows
+    """
+
+    def path(self, stream_slice: Mapping[str, Any] = None, **kwargs) -> str:
+        return f"repos/{stream_slice['repository']}/actions/workflows"
+
+    def parse_response(self, response: requests.Response, stream_slice: Mapping[str, Any] = None, **kwargs) -> Iterable[Mapping]:
+        response = response.json()
+        response["repository"] = stream_slice["repository"]
+        yield response
+
+
+class WorkflowRuns(GithubStream):
+
+    """
+    Get all workflows of a GitHub repository
+    API documentation: https://docs.github.com/en/rest/reference/actions#list-workflow-runs-for-a-repository
+    """
+
+    def path(self, stream_slice: Mapping[str, Any] = None, **kwargs) -> str:
+        return f"repos/{stream_slice['repository']}/actions/runs"
+
+    def parse_response(self, response: requests.Response, stream_slice: Mapping[str, Any] = None, **kwargs) -> Iterable[Mapping]:
+        response = response.json()
+        response["repository"] = stream_slice["repository"]
+        yield response
+
+
+# Security streams
+
+
+class RepositoryVulnerabilityAlert(GithubStream):
+    """
+    Get the dependabot alerts of a GitHub repository
+    API documentation: https://docs.github.com/en/graphql/reference/objects#repositoryvulnerabilityalert
+    """
+
+    @property
+    def http_method(self) -> str:
+        return "POST"
+
+    def path(self, stream_slice: Mapping[str, Any] = None, **kwargs) -> str:
+        return "graphql"
+
+    def request_body_json(
+        self,
+        stream_state: Mapping[str, Any],
+        stream_slice: Mapping[str, any] = None,
+        next_page_token: Mapping[str, Any] = None,
+    ) -> MutableMapping[str, Any]:
+        body = (
+            '{ repository(name: "'
+            + stream_slice["repository"].split("/")[1]
```

Comment:
@bzAmin can you give a variable name for what the index [0] and [1] are? like:
```python
repo = stream_slice["repository"].split("/")[1]
org... [0]?
```
The goal is to make this clear to anyone looking into this code.

Reply:
Both has been put in variables